marklogic-data-hub-central/ui/e2e/cypress/integration/curation/load/defaultIngestion.spec.tsx

```diff
@@ -56,6 +56,7 @@ describe("Default ingestion ", () => {
     loadPage.stepNameInput().should("be.disabled");
     loadPage.stepDescriptionInput().clear().type("UPDATE");
     loadPage.saveButton().click();
+    cy.waitForAsyncRequest();
```

Comment:
I still don't think this function does anything. Navigating through the UI can be done using selectors that are based off of UI state changes. The cypress-wait-until library is a better option.

Reply:
Thanks, I removed the instances that I added and didn't see any negative impact.

Reply:
I reverted on account of e2e failures on the CI/CD side. e2e tests now pass.